### PR TITLE
add: validator for setdress objects names

### PR DESCRIPTION
**Pull request by BlueLag**
## Changelog Description
Add a validator in the publish setdress. The validator check the objects names to raise an error when there is a special character in it's name.

## Testing notes:
1. Open a setdress scene in blender with OpenPype (ex: woollywoolly/Assets/Environement/ChampFleurs_038_A_JOUR --> Fabrication)
2. Open publish the OpenPype tab and wait for the collect step to end (you'll maybe have to relink libraries and clear OP containers with WW Util if the collect step raise an error)
3. Click on the debug button to execute the validate step
4. If there is an error on setdress objects names with a special character except "_" and "." the "Validate SetDress Objects Name" step should raise an error with the name of the object else no error is raised.

### CODE REVIEW

File: openpype/hosts/blender/plugins/publish/validate_setdress_object_names.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import bpy
+import string
+
+import pyblish.api
+
+import openpype.hosts.blender.api.action
+from openpype.pipeline.publish import ValidateContentsOrder
+
+
+class ValidateSetdressObjectNames(pyblish.api.InstancePlugin):
```
Comment (Tilix4) [inline openpype/hosts/blender/plugins/publish/validate_setdress_object_names.py]: Isn't `pyblish.api.Validator` better use here?

**Review by Tilix4 — COMMENTED** [review]
Very nice and pretty clean!

File: openpype/hosts/blender/plugins/publish/validate_setdress_object_names.py (hunk 1 above)
Comment (BlueLag) [reply openpype/hosts/blender/plugins/publish/validate_setdress_object_names.py]: I used an existing validator script from kaamaurice to create mine so I kept `pyblish.api.InstancePlugin`
 
But I'll test with `pyblish.api.Validator` if it works fine ! :)

**Comment by BlueLag** [comment]
It works fine with `pyblish.api.Validator` so here is the change !

**Review by Tilix4 — APPROVED** [review]

**Pull request merged**